client/components/head/index.jsx

```diff
@@ -25,6 +25,15 @@ const Head = ( { title = 'WordPress.com', faviconURL, children, cdn, branchName
 			<meta name="theme-color" content="#0087be" />
 			<meta name="referrer" content="origin" />
 
+			<link
+				rel="prefetch"
+				as="document"
+				href="https://public-api.wordpress.com/wp-admin/rest-proxy/?v=2.0"
+			/>
+			<link rel="preconnect" href="https://s1.wp.com" />
```

Comment:
These all look good 👍 This is the easiest way of saving some roundtrips without having the maintenance burden of keeping up with the exact URLs needed as external resources change.

Reply:
This one is for content that's coming from our CDN, but only for one host. I think we also have `s0` and `s2`, but I'm not sure we really use them now that we're mostly on http/2 / spdy.

Reply:
```suggestion
```